Update Expose Function tutorial (#8945)
kyma-project/kyma: docs/serverless/08-02-expose-function.md

1. Select a Namespace from the drop-down list in the top navigation panel. Make sure the Namespace includes the Function that you want to expose through an API Rule.

2. Go to the **Functions** view in the left navigation panel and select the Function you want to expose.

3. Switch to the **Configuration** tab and select **Expose Function** in the **API Rules** section. A modal with the form will appear.

4. In the **General settings** section:

    - Enter the API Rule's **Name** matching the Function's name.

    >**NOTE:** The APIRule CR can have a different name than the Function, but it is recommended that all related resources share a common name.

    - Enter **Hostname** to indicate the host on which you want to expose your Function.

> 5. In the **Access strategies** section, leave the default settings, with `GET`, `POST`, `PUT`, `DELETE`, `PATCH` and `HEAD` methods and the `noop` handler selected.

Reviewer: ```suggestion
5. In the **Access strategies** section, leave the default settings with `GET`, `POST`, `PUT`, `PATCH`, `DELETE`, and `HEAD` methods and the `noop` handler selected.
```

> 6. Select **Create** to confirm changes. The modal with form will close.

Reviewer: ```suggestion
6. Select **Create** to confirm changes. The pop-up box with the form will close.
```

> 7. Check if you can access the Function by selecting the HTTPS link under **Host** column for created API Rule.

Reviewer: ```suggestion
7. Check if you can access the Function by selecting the HTTPS link under the **Host** column for the newly created API Rule.
```
